cdap-data-fabric/src/main/java/io/cdap/cdap/data2/dataset2/lib/partitioned/PartitionedFileSetDataset.java

```diff
@@ -625,37 +594,14 @@ public void dropPartition(PartitionKey key) {
     }
   }
 
-  private void dropPartitionFromExplore(PartitionKey key) {
-    if (exploreEnabled) {
-      ExploreFacade exploreFacade = exploreFacadeProvider.get();
-      if (exploreFacade != null) {
-        try {
-          exploreFacade.dropPartition(datasetInstanceId, spec, key);
-        } catch (Exception e) {
-          throw new DataSetException(String.format(
-            "Unable to drop partition for key %s from explore table.", key.toString()), e);
-        }
-      }
-    }
-  }
-
   @ReadWrite
   @Override
   public Future<Void> concatenatePartition(PartitionKey key) {
```

Comment:
This method has little sense now, it does not do any mutation anymore. Please remove